api/src/main/resources/workbench-api.yaml

```diff
@@ -7711,3 +7729,60 @@ definitions:
         description: Datetime at creation of this user's accont
         type: string
         format: date-time
+  ReportingResearcher:
```

Comment:
Which elements are required for each of these structures?  We can add that in a `required` section.

Reply:
This isn't part of the API at all, though all environments will be saddled with it. I want to replace it with Immutables or maybe Kotlin data classes so we don't have to put it here.